CommcareTranslationChecker/CommcareTranslationChecker.py

```diff
@@ -140,6 +148,82 @@ def convertCellToOutputValueList(cell):
     return outputList, messages
 
 
+def get_unique_words(cell):
+    """
+    Remove output tags and return unique words. If 'jr://file/' is found in cell return empty list
+    
+    Input:
+    cell (xl.cell.cell.Cell): Cell from which unique words needs to be computed
+
+    Returns:
+    List of unique words present in the cell
+    """
+    text = str(cell.value)
+    openTag = "<output value=\""
+    closeTag = "\"/>"
+    if 'jr://file/' in cell.value:
+        return []
+    output_list, messages = convertCellToOutputValueList(cell)
+    for output_value in output_list:
+        if 'ILL-FORMATTED TAG :' in output_value:
+            return []
+        text = text.replace(f'{openTag}{output_value}{closeTag}', '')
+    words = list(set(re.findall(r'\w+', text)))
+    return words
+
+
+def is_english_word(word):
+    """
+    Checks whether a word contains only english alphabet or not
+    """
+    if len(re.findall('[a-zA-z0-9]', word)) == len(word):
+        return True
+    else:
+        return False
+
+
+def english_translation_mismatch(source_cell, current_cell):
+    """
+    If there are shared english words between source cell and current cell,
+    and if other words in current cell are not in english, return mismatch as true
+    
+    Input:
+    source_cell (xl.cell.cell.Cell): Source cell which contains text of base language
+    target_cell (xl.cell.cell.Cell): Current cell which contains text of translated language
+
+    Returns:
+    shared_english_words(list): list of shared english words between two cells
+    mismatch(boolean): True indicates mismatch, False indicates no mismatch
+    """
+    source_cell_words = get_unique_words(source_cell)
+    current_cell_words = get_unique_words(current_cell)
+    shared_words = set(source_cell_words).intersection(set(current_cell_words))
```

Comment:
do we need `get_unique_words` to return a list? we can just have that return a set and hence then not need to convert it to set again?